maven-model-builder/src/test/resources/poms/factory/complexEmptyAndMissing.xml

```diff
@@ -31,21 +31,5 @@
                 <profile.file>activated-1</profile.file>
             </properties>
         </profile>
-        <profile>
-            <id>another-two-conditions</id>
-            <activation>
-                <property>
-                    <name>myproperty</name>
-                    <value>test</value>
-                </property>
-                <file>
-                	<exists>simple.xml</exists>
-                    <missing>simple.xml</missing>
-                </file>
-            </activation>
-            <properties>
-                <profile.miss>activated-2</profile.miss>
-            </properties>
-        </profile>
     </profiles>
```

Comment:
Can you tell me why you have removed those part of the file and changed the id's ? I don't see any good reason to remove those parts and renaming the id's ?